tools/ci_build/github/azure-pipelines/bigmodels-ci-pipeline.yml

```diff
@@ -115,7 +115,7 @@ stages:
               set -ex; \
               env; \
               ccache -s; \
-              /opt/python/cp310-cp310/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
+              /opt/python/cp38-cp38/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
```

Comment:
This is in docker.  Do you have to revert it?

Reply:
Let me try

Reply:
Oh, `python3` from below are python 3.8

Reply:
Our host machine even doesn't have CUDA. So all GPU tests are in docker.